Cargo.toml

```diff
@@ -21,6 +21,7 @@ include = [
 
 [dependencies]
 bytes = "1"
+bytes-utils = "0.1"
```

Comment:
We have a buflist type already in hyper https://github.com/hyperium/hyper/blob/master/src/common/buf.rs#L6 so we shouldn't need to have to add this extra dep.